spec/factories/works.rb

```diff
@@ -13,6 +13,14 @@
     collection
   end
 
+  trait :published do
+    published_edtf { '2020-02-14' }
+  end
+
+  trait :with_creation_dates do
+    created_edtf { '2020-03-04/2020-10-31' }
```

Comment:
This is A-OK. I'm mostly curious why you chose to do this as a trait instead of defaulting this above. To facilitate testing works w/o creation dates?

Reply:
Sounds good